common/web/keyboard-processor/build.sh

```diff
@@ -0,0 +1,91 @@
+#!/usr/bin/env bash
+#
+# Compile KeymanWeb's 'keyboard-processor' module, one of the components of Web's 'core' module.
+#
+set -eu
+
+## START STANDARD BUILD SCRIPT INCLUDE
+# adjust relative paths as necessary
+THIS_SCRIPT="$(greadlink -f "${BASH_SOURCE[0]}" 2>/dev/null || readlink -f "${BASH_SOURCE[0]}")"
+. "$(dirname "$THIS_SCRIPT")/../../../resources/build/build-utils.sh"
+. "$KEYMAN_ROOT/resources/shellHelperFunctions.sh"
+## END STANDARD BUILD SCRIPT INCLUDE
+
+# This script runs from its own folder
+cd "$(dirname "$THIS_SCRIPT")"
+
+action=
+function action_failure() {
+  if [ -n "$action" ]; then
+    builder_report failure $action
+  fi
+}
+
+trap action_failure err
+
+################################ Main script ################################
+
+c1="${COLOR_BLUE:=<}"
+c0="${COLOR_RESET:=>}"
+
+builder_describe \
+  "Compiles the web-oriented utility function module." \
+  configure clean build test "--ci,-c    For use with ${c1}test${c0} action - emits CI-friendly test reports"
```

Comment:
```suggestion
  configure clean build test "--ci    For use with ${c1}test${c0} action - emits CI-friendly test reports"
```

I don't think we need `-c` as a shorthand for `--ci` -- it's pretty short already and `-c` is somewhat confusing? Especially as I like for all CI to use long-form parameters as a self-documentation step.